### ISSUE TITLE
using visible with singleton

### ISSUE THREAD

**Issue by galacticQuasar** [atomiks/tippyjs-react#196]
Visible doesn't seem to affect a singleton tippy, when it is used the tippy will never be shown

an example sandbox https://codesandbox.io/s/focused-leavitt-xhmnp

**Comment by atomiks** [comment]
I don't think this was ever supported to be honest 🤔 after all, we don't know what content to use if you put `visible: true` to begin with.

**Comment by galacticQuasar** [comment]
I have also tried with setting visible on the reference element tippy , but that doesn't work either

**Comment by atomiks** [comment]
But how would it work if you have two `<Tippy />`s part of the singleton both with `visible: true`? Which takes precedence? There are quite a few intrinsic problems with it I think.

Can I ask what you're using it for?

**Comment by galacticQuasar** [comment]
I'm using it for an option popup for each element in a list of items. 
I had hoped the Singleton would make it easier as only one popup should be open at a time

**Comment by atomiks** [comment]
Sure, I think we're just missing an API or behavior that will make this work. If you can think of something feel free to propose a potential solution

**Comment by galacticQuasar** [comment]
The easiest way I can think of manually handling what child of a singleton would be with a ref to the target tippy, and a ref of null would be invisible.

 Is that something that would be feasible to add?

**Comment by atomiks** [comment]
I think it should be, I'll investigate it soon 😇
